style.css

```diff
@@ -1,18 +1,33 @@
-body {
-    padding: 10px 15px;
-    display: flex;
+.header {
+    font-weight: bold;
+    padding-left: 10px;
+    padding-top: 5px;
+    padding-bottom: 5px;
+    background: #07889B;
+    color: #FFF;
+}
```

Comment:
we should have consistent spacing, blank line between each section or none for any